.github/pull_request_template.md

```diff
@@ -19,6 +19,20 @@
   the functionality of your change
 -->
 
+## Changelog
+<!--
+Add each of the changelog entries as a line item in this section. e.g.
+- fix: Updates the graph
+- feat: Adds a new feature
+
+If this change does not need to added to the changelog, just add a single `skip` line e.g.
+- skip
+
+Valid prefixes: breaking, chore, deprecate, doc, feat, fix, infra, refactor, test
+
+Descriptions following the prefixes must be 50 characters or less
```

Comment:
```suggestion
<!--
Add a short but concise sentence about the impact of this pull request. Prefix an entry with the type of change they correspond to: breaking, chore, deprecate, doc, feat, fix, infra, refactor, test.

- fix: Update the graph
- feat: Add a new feature
```

Reply:
Resolved! I also change the text for:

```Descriptions following the prefixes must be 50 characters or less```

to

```Descriptions following the prefixes must be 100 characters long or less```